pyuvdata/uvdata.py

```diff
@@ -2951,6 +2951,82 @@ def write_miriad(self, filepath, run_check=True, check_extra=True,
                                 clobber=clobber, no_antnums=no_antnums)
         del(miriad_obj)
 
+    def read_mwa_corr_fits(self, filelist, axis=None, use_cotter_flags=False,
+                           correct_cable_len=True, phase_data=True,
```

Comment:
I'd be inclined to default phasing to False. I'm a little reluctant to make changes to the input data unless the user specifically asks for it.

Reply:
I also think we should add code to the `read` method to read these files, but there are significant changes coming to the read method in another PR (#705), so lets hold off. If that PR is merged first, we should add them here, if this is merged first we should add them in that PR.